client/src/UpdateRating.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+
+export default function UpdateRating({ video }) {
+  const [counter, setCounter] = useState(video.rating);
+
+  const handleUpdate = async () => {
+    try {
+      const response = await fetch(
+        `https://kristinadudnyk-fullstack-project.onrender.com/video/${video.id}`,
```

Comment:
I've noticed you've put in this URL a few times, and that in each of them you have a localhost:4500 version commented out.

Can you think of a way to make it easier for you to change which server your frontend is talking to for when you want to test things locally?

Reply:
A brainy mate suggested that I can use 
```
ssl: {
    rejectUnauthorized: false,
  },
```
which makes it run locally